ckanext/honduras/fanstatic/js/main.js

```diff
@@ -0,0 +1,7 @@
+$(window).on('load',function(){
+
+    setTimeout(function(){
+        $('body').find('.select2-container').removeClass("form-control");
```

Comment:
Any reason this can't be removed directly in the relevant template?

Reply:
@amercader i tried to look for that, but,
weirdly by ckan default schema, the select2 tags are working normally without class,  it's just that it adds the class ".form-control" when it picks up the select2 inputs from the schema.